app/src/main/java/ch/sdp/vibester/activity/GameActivity.kt

```diff
@@ -44,7 +44,7 @@ open class GameActivity : AppCompatActivity() {
     }
 
     fun decreaseBarTimer(myBar: ProgressBar) {
```

Comment:
```suggestion
/**
* MISSING DOCS !!!!
*/
    fun decreaseBarTimer(myBar: ProgressBar) {
```

Reply:
That's just a bugfix, not a function I've written. The same class has multiple other public methods that can be documented, however I would rather not write things that might be incorrect. Noting down the missing docs comment however, in case we look at it again in the future.